LICENSE.txt

```diff
@@ -0,0 +1,204 @@
+```
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
```

Comment:
why are we adding this?
copybara must have been updated to and it is no longer excluding this file. This is for internal usage only. There is a license already

Reply:
Removed.